confFiles/default.yaml

```diff
@@ -12,8 +12,12 @@
 #   4) Rand-element - shuffle one of the data item in the output of the target op into random value
 #   5) bitFlip-element - single bit-flip over one data item in the output of the target op
 #   6) bitFlip-tensor - single bit-flip over all data items in the output of the target op 
+#   7) multiBitFlip - multiple bit-flips over all data items in the output of the target op
 ScalarFaultType: None
-TensorFaultType: bitFlip-element 
+TensorFaultType: multiBitFlip
+
+# Number of bits to flip for each data item
+BitCount: 0
```

Comment:
Did you test with different values of BitCount (> 0) and see the results for any model?